### PR TITLE
Prepare frontend

### LINKED ISSUE #27: Prepare the project to have UI
### Goal
The project displays a basic page with the project name and a link to the playground

### Context
We eventually want to have basic UI for user sign in and token generation, show documentation and graphql/sql playgrounds. 
But I also assume that we want to keep API codebase independent from the UI and keep it optional / modular. So we either need to create a separate frontend repo or turn this repo into monorepo. We also need to setup a project to work with modern frontend components (eg vue).

### Tasks
- [x] Propose concrete solution
- [x] Change structure of the project to be suitable for the frontend
- [x] Setup frontend
- [x] Setup proxy/middleware to serve both UI, API and playground
- [x] Make sure the readmes are updated accordingly 
- [x] Make sure the goal is reached

### PR DESCRIPTION
**Pull request by valiafetisov**
Closes #27 

Screenshot 2023-03-29 at 12 00 37

### CODE REVIEW

**Comment by valiafetisov** [comment]
- Please follow newly created [quick start instructions](https://github.com/makerdao-ses/switchboard-boilerplate/blob/0fd3133f6b0094bfdc429b071e009a90fe3cc593/README.md#quick-start) to functionally test it
- Please also review readme text

**Review by LukSteib — CHANGES_REQUESTED** [review]
- all three readmes look good ✔️ 
- only starting frontend works ✔️  (of course gives error when trying to access API playground)
- [x] Tried to start the whole project via `docker-compose up --build --force-recreate` but `http://localhost:3000/` can't be reached (is it possible that frontend and API playground are conflicting here?)

**Review by LukSteib — APPROVED** [review]

File: frontend/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+- [Nuxt 3](https://nuxt.com)
+- Tailwind CSS
+- Naive UI
+- GitHub Actions based CI
+
+## How to get going?
+
+This is a straight-forward setup with minimal templating and scaffolding. The options you selected during the sidebase CLI setup are all here though. Good places to continue reading are:
+- [the First Steps documentation](https://sidebase.io/sidebase/usage)
+- [our discord](https://discord.gg/auc8eCeGzx)
```
Comment (KirillDogadin-std) [inline frontend/README.md:14]: I'm not sure if this was copied over, but seems like term "our discord" in an opensource project for maker infrastructure might cause confusion

**Review by KirillDogadin-std — CHANGES_REQUESTED** [review]

File: frontend/README.md (hunk 1 above)
Comment (valiafetisov) [reply frontend/README.md]: Thanks for spotting it, I rewrote the whole readme now

File: docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    expose:
+      - 3000
+    env_file:
+      - ./api/.env
+    command: >
+      sh -c "
+      cd /app &&
+      pwd &&
+      npx prisma db push &&
+      npx prisma generate &&
```
Comment (KirillDogadin-std) [inline docker-compose.yaml:18]: https://www.prisma.io/docs/concepts/components/prisma-migrate/db-push

> 2. By default, after changes have been applied to the database schema, generators are triggered (for example, Prisma Client). You do not need to manually invoke prisma generate.

File: docker-compose.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    build:
+      context: ./api
+    expose:
+      - 3000
+    env_file:
+      - ./api/.env
+    command: >
+      sh -c "
+      cd /app &&
+      pwd &&
```
Comment (KirillDogadin-std) [inline docker-compose.yaml:16]: is this for debugging purpose? anything else?

**Review by KirillDogadin-std — CHANGES_REQUESTED** [review]

**Pull request merged**